scripts/patterns-reorg/page_content.py

```diff
@@ -73,7 +73,7 @@ def optimize_content(index: str) -> str:
         """\
 ---
 title: Optimize for target hardware
-description: TODO - 50+ characters long to ignore metadata check for now!
+description: Optimize abstract circuits and operators so they can run on quantum hardware. 
```

Comment:
```suggestion
description: Optimize abstract circuits and operators so that they can run on quantum hardware. 
```

Reply:
"that" is optional in the above sentence, but you can add it if you want to.